resources/sync_gateway_configs/custom_sync/wake_changes_access_cc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "interface":":4984",
+    "adminInterface": "0.0.0.0:4985",
+    "maxIncomingConnections": 0,
+    "maxCouchbaseConnections": 16,
+    "maxFileDescriptors": 90000,
+    "slowServerCallWarningThreshold": 500,
+    "compressResponses": false,
+    "log": ["CRUD+", "Cache+", "HTTP+", "Changes+"],
+    "verbose":"true",
+    "databases":{
+        "db":{
+            "server":"http://{{ couchbase_server_primary_node }}:8091",
+            "bucket":"data-bucket",
+            "sync":
+            `function(doc, oldDoc){
+                if(doc._id == "access_doc_0") {
+                    console.log("granting_access!");
+                    access(["adam", "traun", "andy"], "NATGEO");
+                }
```

Comment:
Using a hard coded access grant isn't the end of the world for this specific test, but as you build out more access-based tests I'm guessing you'll want to change this to something dynamic, like you've got for channel handling. e.g.

```
 if(doc.accessUsers && doc.accessChannels) {
    console.log("granting_access!");
    access(doc.accessUsers, doc.accessChannels);
 }